altos-rust/port/cortex-m0/src/peripheral/usart/mod.rs

```diff
@@ -132,26 +314,34 @@ pub fn init() {
     pa2.set_pull(gpio::Pull::Up);
     pa3.set_pull(gpio::Pull::Up);
 
-    let usart2 = Usart::new(UsartX::Usart2);
+    // Creates a Usart2 object and initializes it for serial communication.
+    let mut usart2 = Usart::new(UsartX::Usart2);
+    // Disable Usart prior to initializing.
     usart2.disable_usart();
+
+    // Set word length to Eight
```

Comment:
We really don't need a comment above every statement... Especially considering the function call names